CMake/add_project_fletch.cmake

```diff
@@ -60,6 +60,10 @@ ExternalProject_Add(fletch
     -Dfletch_ENABLE_log4cplus:BOOL=ON
     -Dfletch_ENABLE_shapelib:BOOL=OFF
     -Dfletch_PYTHON_MAJOR_VERSION:STRING=2
+<<<<<<< HEAD
+    -Dfletch_BUILD_INSTALL_PREFIX:STRING=${CMAKE_INSTALL_PREFIX}
+=======
+>>>>>>> refs/heads/master
```

Comment:
This looks like an unresolved merge

Reply:
Thanks! for some reason it is right on my machine... will fix ASAP!